bot/cogs/base_converter_cog.py

```diff
@@ -0,0 +1,82 @@
+import discord
+import discord.ext.commands as commands
+
+from bot.consts import Colors
+
+class BaseConverterCog(commands.Cog):
+
+    def __init__(self, bot) -> None:
+        self.bot = bot
+
+    @commands.group(pass_context= True, invoke_without_command= True, aliases= ['convert','baseconvert'])
+    async def bconvert(self, ctx, *, number) -> None:
+        """
+        A simple base converter that takes in a base number an a value, then displays the value in binary, octal, decimal, and hexadecimal
```

Comment:
good doc comment

Reply:
Thanks! I went back and forth tweaking it a lot.